README.md

```diff
@@ -4,7 +4,7 @@
 ![GitHub](https://img.shields.io/github/license/GenericMappingTools/dcw-gmt)
 
 This repository contains the data and scripts that maintain and build
-the dcw-gmt package used by **GMT 5 or later**.  **Note**: DCW 2.0.0
+the dcw-gmt package used by **GMT 5 or later**.  **Note**: DCW 2.0.1
 requires GMT 6.1.1 or later.
```

Comment:
```suggestion
the dcw-gmt package used by following GMT versions:

| DCW Version | Minimum GMT Version |
|-------------|---------------------|
| DCW 1.1.4   | 5 |
| DCW 2.0.0   | 6.1.1 |
| DCW 2.0.1   | 6.3.0 |
| DCW 2.0.2   | 6.1.1 |
```
I'm not sure if there's a specific GMT 5 version to list for DCW 1.1.4.

Reply:
I think it cannot read 2.* though, so probably OK to state 1.1.4 here.